search_evaluation_framework/create-query-set.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+# Create a query set using sampling.
+docker compose run search_evaluation_framework java -jar /app/search-evaluation-framework.jar -o http://opensearch:9200 -s /app/files/create-query-set.json
```

Comment:
I think it would make sense to provide an example of that file to have the command in the script executable.
Perhaps additional `ADD` instructions for the `create-query-set.json` and `run-query-set.json` files?

Reply:
That file should be mounted by the docker compose.

```
  search_evaluation_framework:
    container_name: search_evaluation_framework
    build: ./search_evaluation_framework
    volumes:
      - ./search_evaluation_framework:/app/files
```

So the json files referenced by these commands are the ones under the `./search_evaluation_framework` directory alongside the bash scripts.

Reply:
I don't see those json files in here!

Reply:
Ah! *.json is git ignored.

Reply:
There we go. Now they're there.